mentor_classifier/mentor_classifier/arch/lr/predict.py

```diff
@@ -125,18 +128,12 @@ def __get_prediction(self, w2v_vector):
             )
         return answer_id, answer_text, answer_media, highest_confidence
 
-    def __get_offtopic(self):
+    def __get_offtopic(self) -> AnswerIdTextAndMedia:
         try:
-            i = random.randint(
-                0, len(self.mentor.utterances_by_type["_OFF_TOPIC_"]) - 1
-            )
-            return (
-                self.mentor.utterances_by_type["_OFF_TOPIC_"][i][0],
-                self.mentor.utterances_by_type["_OFF_TOPIC_"][i][1],
-                self.mentor.utterances_by_type["_OFF_TOPIC_"][i][2],
+            id, text, media = random.choice(
+                self.mentor.utterances_by_type["_OFF_TOPIC_"]
             )
+            return (id, text, media)
+
         except KeyError:
-            return (
-                "_OFF_TOPIC_",
-                "_OFF_TOPIC_",
-            )
+            return ("_OFF_TOPIC_", "_OFF_TOPIC_", "")
```

Comment:
I think this is a problem...don't we need the off-topic media? maybe we have to get it from graphql?

Reply:
Should be [], not "" for an empty array of media (i.e. no media)

Reply:
And doesn't that line only happen if the user doesn't have a single OFF_TOPIC response? in which case you wouldn't be able to get the media from graphql anyway because it doesn't exist at all right?

Reply:
i guess that makes sense...what would the client do in that case?

Reply:
ok I updated the tests based on what you said